### PR TITLE
Nds 499 code inline error

### PR DESCRIPTION
**Pull request by nikola-nulogy**
### What I did

- Added Inline validation component and storybook story

### How to test

- Run Inline Validation story in storybook

I also created clean-up ticket and tickets for designing and building list component

### CODE REVIEW

**Review by clemensp — CHANGES_REQUESTED** [review]

File: docs/src/markdown/get-started/get-started.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: "Get started"
 intro: "Learn how to start designing and developing using the Nulogy Design System"
 ---
 
-![Nulogy logos](logos.png)
+s](logos.png)
```
[the remark this answers is not in the document]
Comment (nikola-nulogy) [reply docs/src/markdown/get-started/get-started.md]: No, it wasn't and it is fixed now.

File: docs/src/markdown/get-started/get-started.md [hunk 2]
```diff
@@ -29,6 +29,7 @@ Congratulations, you've installed the design system! Please see the READMEs link
 <div style="margin-bottom: 64px;"></div>
 
 # Designers
+
```
[the remark this answers is not in the document]
Comment (nikola-nulogy) [reply docs/src/markdown/get-started/get-started.md]: I guess so and it is fixed now.

File: components/src/Validation/InlineValidation.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+import Flex from "../Flex/Flex";
+import Box from "../Box/Box";
+import icons from "../../icons/icons.json";
+
+export const iconNames = Object.keys(icons);
+
+const InlineValidation = props => (
+  <Box color={ props.color }>
+    <Flex my={ 2 }>
+      <Icon size={ theme.space[4] } name={ props.icon } mr={ 1 } />
```
Comment (clemensp) [inline components/src/Validation/InlineValidation.js]: Do we want to try the `icon={ {name: "error"} }` approach here?

File: components/src/Icon/Icon.js [hunk 4]
```diff
@@ -33,10 +33,11 @@ Svg.propTypes = {
   name: PropTypes.oneOf(iconNames).isRequired,
   size: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
   title: PropTypes.string,
-  color: PropTypes.string.isRequired,
+  color: PropTypes.string,
 };
 
 Svg.defaultProps = {
+  color: "currentColor",
```
Comment (clemensp) [inline components/src/Icon/Icon.js:40]: I took a look at the `InlineIcon` proptypes, and it looks like they're not needed anymore.
Could you remove those?

**Review by clemensp — APPROVED** [review]
Changes look good.

Can you address the one comment I left?
After that, looks good to merge.

**Pull request merged**